source/manual/create-a-gpg-key.html.md

```diff
@@ -128,4 +107,11 @@ so to encrypt your GPG key you'll use:
 
 If your GPG key has expired, the easiest way to update it is to [extend the expiry date](https://superuser.com/questions/813421/can-you-extend-the-expiration-date-of-an-already-expired-gpg-key).
 
-You may also find that you have sub-keys that are due to expire too. If you are using a Mac, the easiest way to check is to use the GPG Keychain application. In GPG Keychain, double-click on your key to see the details and go the the "Subkeys" tab. If you have a sub-key that is due to expire, you'll be asked if you want to update it and push the key to the keyserver.
+You may also find that you have sub-keys that are due to expire too. If you are using a Mac, the easiest way to check is to use the GPG Keychain application. In GPG Keychain, double-click on your key to see the details and go the the "Subkeys" tab. If you have a sub-key that is due to expire, you'll be asked if you want to update it.
+
+After extending (or renewing) your GPG key, you will need to extract the public key and raise a PR to update `govuk_secrets`.
+
+```
+gpg --export -a --output ~/govuk/govuk-secrets/gpg_public_keys/your_name.asc your.email@digital.cabinet-office.gov.uk
```

Comment:
Can this command be moved to the `## Save your public key to `govuk-secrets` section? It looks nice and idempotent, whereas the earlier one looks more manual (you have to move said key somewhere).

Then this section can just refer to it, i.e. `[extract the public key](#save-your-public-key-to-govuk-secrets)`.

Reply:
That's a good idea.  I'll update this.